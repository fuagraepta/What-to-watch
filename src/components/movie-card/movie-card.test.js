import React from 'react';
import renderer from 'react-test-renderer';

import MovieCard from './movie-card.jsx';

const mockComponent = <div>Hello World</div>;


describe(`The component is rendered correctly`, () => {
  it(`Movie card correctly renders with mock components`, () => {
    const movieCard = renderer
      .create(
          <MovieCard>
            {mockComponent}
          </MovieCard>
      )
      .toJSON();

    expect(movieCard).toMatchSnapshot();
  });
});
