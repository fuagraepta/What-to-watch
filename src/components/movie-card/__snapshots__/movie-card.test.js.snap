// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component is rendered correctly Movie card correctly renders with mock components 1`] = `
<section
  className="movie-card "
>
  <div>
    Hello World
  </div>
</section>
`;
